numerics/elliptic_integrals.cpp

```diff
@@ -45,39 +45,61 @@ double EllipticNomeQ(double mc);
 // where m = 1 - mc.  The method is similar to that described in [Fuk11a].
 void FukushimaEllipticBD(double mc, Angle& B_m, Angle& D_m);
 
+// The functions that compute elliptic integrals of multiple kinds may be called
+// without computing the integral of the third kind.  Passing |unused| bypasses
+// the corresponding code path at compile time, passing a reference to an
+// |Angle| performs the computation.
+
+struct UnusedResult {
+  constexpr UnusedResult(base::uninitialized_t) {}
+};
+
+template<typename T, typename = EnableIfAngleResult<T>>
+inline constexpr bool should_compute = !std::is_same_v<T, UnusedResult const>;
+
+inline constexpr UnusedResult unused{uninitialized};
+
+template<typename T>
+using EnableIfAngleResult =
+    std::enable_if_t<std::is_same_v<T, UnusedResult const> ||
+                     std::is_same_v<T, Angle>>;
```

Comment:
I'd reorder things a bit for ease of readability:
1. `UnusedResult`,
1. `unused`,
1. `EnableIfAngleResult`,
1. `should_compute`.

Reply:
Done. That also makes it legal, MSVC is overly lenient as usual.